tiny_dnn/core/framework/tensor.h

```diff
@@ -278,17 +279,17 @@ auto host_data() {
 
   Tensor operator[](size_t index) { return Tensor(storage_[index]); }
 
-  template <typename S>
-  xt::xrange<S> get_range(std::initializer_list<const S> list) const {
-    // if (*(list.begin()) == -1) return xt::all();
-    // if(*(list.begin())==*(list.begin()+1)) return *(list.begin());
+  /*template <typename S>
```

Comment:
They are really problematic, all kind of type problems, you can check compilation errors. Also I want to use, for example, strided views in future and that would be either overcomplicated or impossible